src/Databases/DatabaseOnDisk.h

```diff
@@ -98,6 +98,24 @@ class DatabaseOnDisk : public DatabaseWithOwnTablesBase
 
     const String metadata_path;
     const String data_path;
+
+/// Daisy: starts.
+public:
+    std::tuple<String, String> getCreateTableQueryAndEngineFullString(const String & table_name, ContextPtr context) const override;
```

Comment:
can it be shorter like getCreateTableFullString(...)?

Reply:
I think even if the function name is shorter, it is still not concise enough. so why not make it easier to understand?
here return **two String**, it seems that it's OK to use 'string And string'.